src/components/select/select.component.spec.ts

```diff
@@ -604,4 +604,108 @@ describe('Select Component - With custom Icon', () => {
     function getCustomIcon(): HTMLElement | null {
         return nativeElement.querySelector('.ux-select-icon');
     }
-});
\ No newline at end of file
+});
+
+@Component({
+    selector: 'app-select-with-recent-options-test',
+    template: `
+        <ux-select [(value)]="value"
+            [options]="options"
+            [multiple]="multiple"
+            [allowNull]="allowNull"
+            [(dropdownOpen)]="dropdownOpen"
+            [(recentOptions)]="recentOptions"
+            [recentOptionsMaxCount]="recentOptionsMaxCount"
+            (recentOptionsChange)="onRecentOptionsChange($event)"></ux-select>
+    `
+})
+export class SelectWithRecentOptionsTestComponent {
+
+    value: string | string[];
+    options: string[] = ['One', 'Two', 'Three'];
+    multiple: boolean = false;
+    allowNull: boolean = false;
+    dropdownOpen: boolean = true;
+    recentOptions: string[] = [];
+    recentOptionsMaxCount = 2;
+
+    onRecentOptionsChange(): void {}
+}
+
+fdescribe('Select with recent options', () => {
+
```

Comment:
Just noticed there is an f left in-front of the describe.

Reply:
Sorry, still making changes on this.